daisy_workflows/image_import/enterprise_linux/translate_centos_8.wf.json

```diff
@@ -43,7 +43,7 @@
       "CreateDisks": [
         {
           "Name": "disk-translator",
-          "SourceImage": "projects/compute-image-tools/global/images/family/debian-9-worker",
+          "SourceImage": "projects/compute-image-tools/global/images/family/debian-10-worker",
```

Comment:
@zoran15 
> Will this not cause the regression that we had in last year when we reverted back to Debian 9?

Good question! I re-ran the affected tests and couldn't repro the issue, so my *guess* is that a package update in the Debian 10 repos fixed the issue.

The previous error while running a shell process for `virt-customize`: https://github.com/GoogleCloudPlatform/compute-image-tools/blob/3a6d1ada541e4f53cb7c92dfdc67c120e81228dd/daisy_workflows/image_import/enterprise_linux/translate.py#L358         

(#1438 has links to the failing e2e tests)